lib/Service/YAPServer.hpp

```diff
@@ -235,16 +233,35 @@ class YAPServer
     }
 
     /**
-     * Get the number of configured channels.
+     * Get the number of configured TX channels.
      * @returns Number of configured Data Channels. The Command Channel is ignored here.
      */
-    uint8_t getNumberOfChannels()
+    uint8_t getNumberOfTxChannels()

```

Comment:
getNumberOfTxChannels clones getNumberOfRxChannels. Do we need the logic or is it easier with less space to use a member variable, counting at registration or successful subscription?